test/src/core/testCreateSession.js

```diff
@@ -568,9 +567,7 @@ describe('test Request With Credentials', function () {
     nock('https://hub.browserstack.com')
       .post('/wd/hub/session')
       .reply(201, function (uri, requestBody) {
-        const reqObj = JSON.parse(requestBody);
```

Comment:
what causes this change? Is it because of the nocks upgrade?

Reply:
Yes